server/athenian/api/openapi/openapi.yaml

```diff
@@ -5657,6 +5701,27 @@ components:
       - name
       - rank
       type: object
+    JIRAStatus:
+      additionalProperties: false
+      description: JIRA issue status details.
+      example:
+        name: Done
+        stage: finished
+      properties:
+        name:
+          description: Exact status name.
+          type: string
+        stage:
+          description: One of the three status categories.
+          enum:
+          - backlog
```

Comment:
The 3 Jira status categories are officially called: To Do, In Progress, Done
see https://athenianco.atlassian.net/secure/admin/ViewStatuses.jspa